### PR TITLE
Added snapScrollTo methods

### PR DESCRIPTION
**Pull request by Nohus**
See #35

### CODE REVIEW

File: mapcompose/src/main/java/ovh/plrapps/mapcompose/api/LayoutApi.kt [hunk 1]
```diff
@@ -56,7 +57,7 @@ val MapState.scroll: Offset
  * suspending call because it's required to wait the first composition. Otherwise, it's invoked
  * immediately.
  * This is a low-level concept (input value is expected to be in scaled pixels). To scroll to a
- * known position, prefer the [scrollTo] API.
+ * known position, prefer the [snapScrollTo] API.
```
Comment (peterLaurence) [inline mapcompose/src/main/java/ovh/plrapps/mapcompose/api/LayoutApi.kt:60]: Perfect, I also wanted to update this part of the doc.

File: mapcompose/src/main/java/ovh/plrapps/mapcompose/api/LayoutApi.kt [hunk 2]
```diff
@@ -157,6 +158,32 @@ suspend fun MapState.rotateTo(
...
+ * @param x The normalized X position on the map, in range [0..1]
+ * @param y The normalized Y position on the map, in range [0..1]
+ * @param screenOffset Offset of the screen relatively to its dimension. Default is
+ * Offset(-0.5f, -0.5f), so moving the screen by half the width left and by half the height top,
+ * effectively centering on the scroll destination.
+ */
+suspend fun MapState.snapScrollTo(
+    x: Double,
+    y: Double,
+    screenOffset: Offset = Offset(-0.5f, -0.5f)
```
Comment (peterLaurence) [inline mapcompose/src/main/java/ovh/plrapps/mapcompose/api/LayoutApi.kt:173]: Good default

**Review by peterLaurence — APPROVED** [review]

**Comment by Nohus** [comment]
I'm afraid I don't have permission to merge it even if you approve. :smiling_face_with_tear:

**Comment by peterLaurence** [comment]
I sent you an invite to be a collaborator.

**Pull request merged**